iset.mm

```diff
@@ -103467,68 +103480,122 @@ Cauchy sequence (for example, fixing a rate of convergence as in
+      cuz wss wdc csb cif cseq wbr w3a wrex cfz co wf1o cle wceq wa wex df-isum
+      wo cio nfss nfcri nfdc nfralxy nfcsb1v nfif nfmpt nfiseq nfbr nf3an nff1o
+      nfrexya nfv nffv nfeq2 nfan nfex nfor nfiotaxy nfcxfr ) CABCUAAEJZUDKZUEZ
+      FJALZUFZFWDUBZMNGOGJZALZCWIBUGZPUHZUCZWCUIZHJZQUJZUKZEOULZRWCUMUNZAIJZUOZ
+      WOWCMNGSWIWCUPUJZCWIWTKZBUGZPUHZUCZRUIZKZUQZURZIUSZESULZVAZHVBHABIFCEGUTX
+      MCHWRXLCWQCEOCOTZWEWHWPCCAWDDCWDTZVCWGCFWDXOWFCCFADVDVEVFCWNWOQCMNWMWCCWC
+      TZCMTZCGOWLXNWJCWKPCGADVDCWIBVGCPTZVHVICNTZVJCQTCWOTVKVLVNXKCESCSTZXJCIXA
+      XICCWSAWTCWTTCWSTDVMCWOXHCWCXGCMNXFRCRTXQCGSXEXTXBCXDPXBCVOCXCBVGXRVHVIXS
+      VJXPVPVQVRVSVNVTWAWB $.
   $}
 
   ${
-    $d f m n z k $.  $d f m n z x $.  $d f m n z A $.  $d f m n z B $.
+    $d j f m n z k $.  $d j f m n z x $.  $d j f m n z A $.  $d j f m n z B $.
     nfsum.1 $e |- F/_ x A $.
     nfsum.2 $e |- F/_ x B $.
     $( Bound-variable hypothesis builder for sum: if ` x ` is (effectively) not
        free in ` A ` and ` B ` , it is not free in ` sum_ k e. A B ` .
        (Contributed by NM, 11-Dec-2005.)  (Revised by Mario Carneiro,
        13-Jun-2019.) $)
     nfsum $p |- F/_ x sum_ k e. A B $=
-      ( vm vn vz vf cv cfv caddc cc cz cc0 cli c1 cn nfcv csu cuz wss wcel cmpt
-      csb cif cseq wbr wa wrex cfz co wf1o wceq wex cio df-sum nfss nfcri nfcsb
-      nfif nfmpt nfiseq nfbr nfan nfrexya nff1o nffv nfeq2 nfex nfiotaxy nfcxfr
-      wo nfor ) ABCDUABGKZUBLZUCZMNHOHKZBUDZDVSCUFZPUGZUEZVPUHZIKZQUIZUJZGOUKZR
-      VPULUMZBJKZUNZWEVPMNHSDVSWJLZCUFZUEZRUHZLZUOZUJZJUPZGSUKZVNZIUQIBCJDGHURX
-      AAIWHWTAWGAGOAOTZVRWFAABVQEAVQTUSAWDWEQAMNWCVPAVPTZAMTZAHOWBXBVTAWAPAHBEU
-      TADVSCAVSTFVAAPTVBVCANTZVDAQTAWETVEVFVGWSAGSASTZWRAJWKWQAAWIBWJAWJTAWITEV
-      HAWEWPAVPWOAMNWNRARTXDAHSWMXFADWLCAWLTFVAVCXEVDXCVIVJVFVKVGVOVLVM $.
+      ( vm vj vn vz vf cv cfv caddc cc cz cc0 c1 cn nfcv csu cuz wcel wral cmpt
+      wss wdc csb cif cseq cli wbr w3a wrex cfz co wf1o cle wceq wa wex df-isum
+      wo cio nfss nfcri nfdc nfralxy nfcsb nfif nfmpt nfiseq nfbr nf3an nfrexxy
+      nff1o nfv nffv nfeq2 nfan nfex nfor nfiotaxy nfcxfr ) ABCDUABGLZUBMZUFZHL
+      BUCZUGZHWFUDZNOIPILZBUCZDWKCUHZQUIZUEZWEUJZJLZUKULZUMZGPUNZRWEUOUPZBKLZUQ
+      ZWQWENOISWKWEURULZDWKXBMZCUHZQUIZUEZRUJZMZUSZUTZKVAZGSUNZVCZJVDJBCKHDGIVB
+      XOAJWTXNAWSAGPAPTZWGWJWRAABWFEAWFTZVEWIAHWFXQWHAAHBEVFVGVHAWPWQUKANOWOWEA
+      WETZANTZAIPWNXPWLAWMQAIBEVFADWKCAWKTFVIAQTZVJVKAOTZVLAUKTAWQTVMVNVOXMAGSA
+      STZXLAKXCXKAAXABXBAXBTAXATEVPAWQXJAWEXIANOXHRARTXSAISXGYBXDAXFQXDAVQADXEC
+      AXETFVIXTVJVKYAVLXRVRVSVTWAVOWBWCWD $.
+  $}
+
+  ${
+    $d A x $.  $d M x $.  $d N x $.
+    sumdc.m $e |- ( ph -> M e. ZZ ) $.
+    sumdc.ss $e |- ( ph -> A C_ ( ZZ>= ` M ) ) $.
+    sumdc.dc $e |- ( ph -> A. x e. ( ZZ>= ` M ) DECID x e. A ) $.
+    sumdc.n $e |- ( ph -> N e. ZZ ) $.
+    $( Restating decidability of a summable set indexed by integers.
+       (Contributed by Jim Kingdon, 1-Jan-2022.) $)
```

Comment:
Even if the current applications of this result are when A is a set of indices, the result itself is about any upperset of integers, so maybe adapt the comment ?
ctually, it might be useful to add to iset.mm a more general result (and maybe also a definition, or else "inline" the definition in the proof): say "A is decidable in B" if ` A. x e. B DECID x e. A `; then, if A is a decidable subclass of B (meaning: it is a subclass of B and it is decidable in B), and B is decidable in C, then A is decidable in C.
The idea of a decidable class (in another) is reminiscent of my definition of bounded class http://us2.metamath.org/ileuni/df-bdc.html

Reply:
I changed the comment to "Decidability of a subset of upper integers."

As for related theorems, I don't have a super clear vision of exactly what to prove in this area - I tend to prove them as I need them. Also in this general family are http://us.metamath.org/ileuni/infssuzcldc.html , http://us.metamath.org/ileuni/infssuzledc.html , http://us.metamath.org/ileuni/zsupcl.html , and perhaps others.